m2stats/NCAindices/seasonal/computeRindices_seas.py

```diff
@@ -215,7 +215,7 @@ def Rindex(doyindex1,doyindex2,pctl,outfile):
 rx1d_var.long_name='maximum one-day precipitation amount'
 
 #Compute consecutive wet days
-wetdays=prectot
+wetdays=prectot+0
```

Comment:
The modifications on 'wetdays' were also affecting 'prectot', leading to issues when 'prectot' is used later to compute other variables. Adding zero seems to make 'wetdays' a new array rather than just pointing to 'prectot'

Reply:
Ah, OK, that makes sense! That's because numpy arrays --- like Python dictionaries --- are not copied on assignment, but rather creating another reference to the same object.

A much better way to clearly communicate this is by explicitly using the `.copy()` method:

```suggestion
wetdays=prectot.copy()
```

https://numpy.org/doc/stable/reference/generated/numpy.ndarray.copy.html